myapp/src/Day3/Assignment3.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
@anantmishra-successive resolve conflicts for this PR

Reply:
![done](https://github.com/anantmishra-successive/React_Training/assets/162098832/8cf8f76e-9ba8-41e1-bd3c-1c1b09ea56b1)
Done Sir !!